.travis.yml

```diff
@@ -166,6 +168,10 @@ script:
   # i.MX6UL 14X14 EVK
   - $make PLATFORM=imx-mx6ulevk
 
+  # i.MX6Quad SABRE
+  - $make PLATFORM=imx-mx6qsabrelite
+  - $make PLATFORM=imx-mx6qsabresd CFG_BOOT_SECONDARY_REQUEST=y
```

Comment:
Should we have `CFG_BOOT_SECONDARY_REQUEST ?= y` set in `plat-imx/conf.mk` for this platform flavor? My goal is to have the most likely configuration built by default, as a general rule.